worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/Flusher.scala

```diff
@@ -185,3 +185,21 @@ final private[worker] class HdfsFlusher(
 
   override def toString: String = s"HdfsFlusher@$flusherId"
 }
+
+final private[worker] class S3Flusher(
```

Comment:
According to this new Flusher. I think this PR is intended to support S3 and HDFS concurrently.